src/test/java/de/tum/in/www1/artemis/connector/GitlabRequestMockProvider.java

```diff
@@ -60,7 +60,7 @@ public class GitlabRequestMockProvider {
     private URL gitlabServerUrl;
 
     @Value("${artemis.version-control.health-api-token}")
```

Comment:
```suggestion
    @Value("${artemis.version-control.health-api-token:#{null}}")
```

Reply:
The test config provides a default value, and the attribute is not specified as `Optional` here. Therefore, I’d prefer to leave it as-is.